source/manual/2nd-line.html.md

```diff
@@ -66,19 +72,11 @@ If you do not work a 5-day week, please talk to your delivery manager to arrange
 
 ## Monitoring
 
-We have a [Technical 2nd Line dashboard][] showing a high level overview of the state of the GOV.UK environments. You can also [install our Chrome extension][] if you want a permanently visible overview. You will need to be connected to the [GDS VPN][gds-vpn] if accessing from home.
-
-### Icinga
-
-We use Icinga to monitor our platform across our environments ([CI/Integration][], [Staging][], [Production][]) and alert us when things go wrong. Many alerts have corresponding documentation in our [GOV.UK developer docs][], detailing how to respond. [Read more about Icinga][].
-
-If you see critical alerts that aren't easily solved, start a thread in the #govuk-2ndline-tech Slack channel.
-
-NB: Technical 2nd Line should investigate these alerts when there is nothing more urgent happening; you do not necessarily have to fix them.
+We have a [Technical 2nd Line dashboard][] showing a high level overview of the state of the GOV.UK environments. You can also [install our Chrome extension][] if you want a permanently visible overview.
```

Comment:
Nit:
```suggestion
```
The dashboard only shows how many zendesk tickets there is and a bunch of links. So I dunno if there's any value sending people there any more.

Reply:
No worries, that's on my list to bump off separately 👍